simplq/src/components/pages/Admin/AdminPage.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState, useEffect, useCallback } from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectTokens, selectQueueName } from 'store/selectedQueue';
+import Ribbon from 'components/common/Ribbon';
+import Tour from 'components/common/Tour';
+import { useGetSelectedQueue } from 'store/asyncActions';
+import HeaderSection from './AdminHeaderSection';
+import TokenList from './TokenList';
+import styles from './admin.module.scss';
+import SidePanel from './AdminSidePanel';
+import getToursteps from './TourSteps';
+
+const TIMEOUT = 10000;
+let timeoutId;
+
+export default (props) => {
+  const queueId = props.match.params.queueId;
+  const queueName = useSelector(selectQueueName);
+  const tokens = useSelector(selectTokens);
+  const description = queueName && 'Ready to share';
+  const dispatch = useDispatch();
+  const getSelectedQueue = useGetSelectedQueue();
+
+  const [toursteps, setToursteps] = useState(getToursteps(window.innerHeight));
+  const { isAuthenticated } = useAuth0();
+
+  const update = useCallback(() => {
+    clearTimeout(timeoutId);
+    dispatch(getSelectedQueue({ queueId }));
+    setTimeout(update, TIMEOUT);
```

Comment:
you need to assign the return to `timeoutId` but I don't really understand what are you trying to do here.